docs/source/setup_osx.rst

```diff
@@ -266,7 +266,7 @@ The JS dependencies are installed using node.js package management (npm).
 
 .. code-block:: bash
 
-    ./bin/install_javascript_dependencies.sh
+    npm install
```

Comment:
thanks for leaving the install_javascript_dependencies.sh file around for legacy sake. I will update the docker file to just call npm install.